### PR TITLE
RSDK-2875 - Update README to work for modules

### PR DESCRIPTION
**Pull request by kkufieta**
Ticket: https://viam.atlassian.net/browse/RSDK-2875

Done:
* Updated README to reflect the changes made during modularization

View the updated README here: https://github.com/kkufieta/viam-cartographer/blob/kk/rsdk-2875-update-readme-makefile/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ Install viam-cartographer:
 
 * Linux aarch64:
     ```bash
-    sudo curl -o /usr/local/bin/carto_grpc_server http://packages.viam.com/apps/slam-servers/carto_grpc_server-stable-aarch64.AppImage
-    sudo chmod a+rx /usr/local/bin/carto_grpc_server
+    sudo curl -o /usr/local/bin/cartographer-module https://storage.googleapis.com/packages.viam.com/apps/slam-servers/cartographer-module-stable-aarch64.AppImage
```
Comment (EshaMaharishi) [inline README.md:20]: Hmm this seems off - there is both storage.googleapis.com and packages.viam.com. Should it remain just packages.viam.com?

Comment (nicksanford) [reply README.md:20]: Yes it should stay package.viam

Comment (kkufieta) [reply README.md:20]: Oh interesting, I copied/pasted this from the docs. Does this mean we should update the docs?

Comment (EshaMaharishi) [reply README.md:20]: oh, yes! i'm surprised these links are considered valid by the browser...

Comment (kkufieta) [reply README.md]: ok fixed!

**Review by EshaMaharishi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (EshaMaharishi) [reply README.md]: thank you!

**Review by nicksanford — APPROVED** [review]

**Pull request merged**